<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualSerialController</class>
 <widget class="QWidget" name="ManualSerialController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>281</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>552</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="byteEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>351</width>
     <height>111</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>77</width>
     <height>257</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="enqButton">
      <property name="text">
       <string>ENQ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ackButton">
      <property name="text">
       <string>ACK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nakButton">
      <property name="text">
       <string>NAK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="eotButton">
      <property name="text">
       <string>EOT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stxButton">
      <property name="text">
       <string>STX</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="etxButton">
      <property name="text">
       <string>ETX</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="etbButton">
      <property name="text">
       <string>ETB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="wackButton">
      <property name="text">
       <string>WACK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rviButton">
      <property name="text">
       <string>RVI</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>77</width>
     <height>262</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="addTextButton">
      <property name="text">
       <string>add text</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="byteBox">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="displayIntegerBase">
       <number>16</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="byteButton">
      <property name="text">
       <string>add byte</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="portName">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openPortButton">
      <property name="text">
       <string>Open port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closePortButton">
      <property name="text">
       <string>Close port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="writeButton">
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="readButton">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
